### PR TITLE
ci(release): Use semantic-release

### LINKED ISSUE #79: Automate release process
It'd probably be worth automating releases with [semantic-release](https://github.com/semantic-release/semantic-release).

### PR DESCRIPTION
**Pull request by markdalgleish**
After this PR is merged, I'll follow up with a dummy patch release to make sure everything works.

closes #79

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -271,14 +271,12 @@ render() {
...
-This package follows [semantic versioning](http://semver.org). In the `master` branch, run `npm version` with the appropriate version bump:
+This package follows [semantic versioning](http://semver.org). New versions are published automatically from [Travis CI](https://travis-ci.org) using [semantic-release](https://github.com/semantic-release/semantic-release).
 
-```bash
-$ npm version major
-$ npm version minor
-$ npm version patch
-```
+In order to automatically increment version numbers correctly, commit messages must follow the [AngularJS commit message convention](https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#-git-commit-guidelines). To make this easier, you can use [commitizen](https://github.com/commitizen/cz-cli) to guide you through the commit process by running `npm run commit` from the command line.
+
```
Comment (jahredhope) [inline README.md]: Do we like badges?
[Commitizen friendly](http://commitizen.github.io/cz-cli/)
`[Commitizen friendly](http://commitizen.github.io/cz-cli/)`

File: .travis.yml [hunk 2]
```diff
@@ -1 +1,16 @@
 language: node_js
+cache:
+  directories:
+    - node_modules
```
Comment (jahredhope) [inline .travis.yml]: This seems like an unrelated change. Should it be coupled to this PR?

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "seek-style-guide",
-  "version": "15.1.0",
+  "version": "0.0.0-development",
```
Comment (mjt01) [inline package.json]: Whats the reason for this version change?

File: .travis.yml [hunk 4]
```diff
@@ -1 +1,16 @@
...
+notifications:
+  email: false
+node_js:
+  - '7'
```
Comment (jahredhope) [inline .travis.yml]: This is going to run two seperate buids?
Does that mean it'll publish twice?

File: README.md (hunk 1 above)
Comment (markdalgleish) [inline README.md]: We sure do 😄

File: .travis.yml (hunk 2 above)
Comment (markdalgleish) [inline .travis.yml]: This was created by the semantic-release setup script.

File: .travis.yml (hunk 4 above)
Comment (markdalgleish) [inline .travis.yml]: No, semantic-release is smart enough to wait for all builds to finish, only publishing from the first build.

File: package.json (hunk 3 above)
Comment (markdalgleish) [inline package.json]: In semantic-release land, you don't check the version number into source control, it becomes an artifact of the build. This is the standard placeholder used instead.

File: README.md (hunk 1 above)
Comment (markdalgleish) [inline README.md]: screen shot 2017-01-17 at 11 46 13 am
